.github/workflows/snyk.yml

```
@@ -0,0 +1,110 @@
+        id: detect-node
+        run: |
+          if [ -f frontend/package.json ]; then
+            echo "found=true" >> $GITHUB_OUTPUT
+          else
+            echo "found=false" >> $GITHUB_OUTPUT
+          fi
+
+      # Run Snyk Node only if Node project present and Snyk is configured
+      - name: Run Snyk to check for vulnerabilities (Node.js)
+        if: steps.detect-node.outputs.found == 'true' && steps.check-snyk.outputs.configured == 'true'
+        working-directory: frontend
```

actionlint (1.7.8)

"working-directory" is not available with "uses". it is only available with "run"

(syntax-check)